lib/virtfs/iso9660/base.rb

```diff
@@ -0,0 +1,81 @@
+require 'rufus-lru'
+require 'virtfs/nativefs/thin'
```

Comment:
To pull in the thin interface, IIUC it's one of the two virtfs interfaces which each module must separately pull in & implement the details.

If this is not required let me know and  I can remove it from the PR & rebase the others